tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Download helm.
   unarchive:
-    src: https://get.helm.sh/helm-{{ helm_version }}-{{ helm_platform }}-{{ helm_arch }}.tar.gz
+    src: https://get.helm.sh/helm-v{{ helm_version }}-{{ helm_platform }}-{{ helm_arch }}.tar.gz
```

Comment:
This seems incorrect — there's already a `v` in the `helm_version`...

Reply:
Hello Jeff

Yes, I mistakenly redefined `helm_version` in another place and it as conflicting.

Thanks for building and supporting so many great roles. I purchased and read your book many times

Best,
Boris